Makefile

```diff
@@ -1,7 +1,7 @@
 # Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 
 SRC_PATH         = $(dir $(realpath $(lastword $(MAKEFILE_LIST))))
-HOST_MACHINE     = $(shell uname -m)
+HOST_MACHINE     = x86_64
```

Comment:
This old setup won't work on macos arm64 from best I can understand. I needed to make the changes in this file in order to get things working on my local machine. Happy to revert the changes here if they are not desired.  

From what I understand, to get reproducible results across docker host platforms, we can pin this to something like x86 and then just make sure to specify the platform when both building and running the docker image.